src/PanelTraits/Boxes.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Boxes
+{
+    public $sideBoxesEnabled = false;
+    public $boxesOptions = [];
+
+    public function enableSideBoxes()
+    {
+        $this->sideBoxesEnabled = true;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function disableSideBoxes()
+    {
+        $this->sideBoxesEnabled = false;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesEnabled()
+    {
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesDisabled()
+    {
+        return ! $this->sideBoxesEnabled;
+    }
+
+    public function boxExists($label)
+    {
+        $boxes = $this->getBoxes();
+
+        return in_array($label, $boxes);
+    }
+
+    public function getLastBox()
+    {
+        $boxes = $this->getBoxes();
+
+        if (count($boxes)) {
+            return last($boxes);
+        }
+
+        return false;
```

Comment:
Good question. This `return false` is inspired from `getLastTab()` from Tabs trait. You're right, return `null` seems to be more coherent here, but i also think we have to be consistent (and change `getLastTab()` too?).